README.md

```diff
@@ -1,17 +1,81 @@
 # Repository *jsramverk-train-controller*
+
 This is the repository for the course *jsramverk* (h23).
 It is maintained by *elmo22* and *poak22*.
 
+The application can be found here: [Train-controller](https://www.student.bth.se/~elmo22/train/). And the backend can be found here: [Backend](https://jsramverk-train-poak22-elmo22.azurewebsites.net/).
+
+## How to use this repo
+
 ```
-# To initialze the application you need to run the following script from root folder in repo.
+# To initialze the application, run the following script from root folder in repo.
 bash setup_app.bash
 ```
 
-You also need a .env file in the /backend folder with API-key, see *.env.example* for structure.
+A a .env file is needed in the /backend folder with API-key, see *.env.example* for structure.
```

Comment:
förbättringsförslag: An .env file is needed in the /backend folder with API-key, see *.env.example* for structure.